bcipy/signal/model/fusion_model.py

```diff
@@ -0,0 +1,114 @@
+from pathlib import Path
+
+import numpy as np
+from bcipy.signal.model import SignalModel
+from sklearn.mixture import GaussianMixture
+from bcipy.helpers.stimuli import GazeReshaper
+from sklearn.model_selection import cross_val_score  # noqa
+from sklearn.utils.estimator_checks import check_estimator  # noqa
+
+
+class GazeModel(SignalModel):
```

Comment:
Following the other signal models, this should likely be a `model/gaussian/gaze_model.py`. Then, when we have a mixture or fusion model, we can define another module named as you have.